### PR TITLE
Refactor: `wrappers` -> `modules`

### LINKED ISSUE #168: Refactor `wrappers` to `modules`

### PR DESCRIPTION
**Pull request by skim0119**
Resolve #168

### CODE REVIEW

**Review by armantekinalp — APPROVED** [review]
LGTM

File: docs/guide/workflow.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ class SystemSimulator(
 ):
     pass
 ```
-This simply combines all the wrappers previously imported together. If a wrapper is not needed for the simulation, it does not need to be added here.
+This simply combines all the modules previously imported together. If a modules are not needed for the simulation, it does not need to be added here.
```
Comment (bhosale2) [inline docs/guide/workflow.md:31]:
```suggestion
This simply combines all the modules previously imported together. If a module is not needed for the simulation, it does not need to be added here.
```

**Review by bhosale2 — CHANGES_REQUESTED** [review]
LGTM mostly, super minor change.

**Review by bhosale2 — APPROVED** [review]
LGTM.

**Pull request merged**